models/index.js

```diff
@@ -37,3 +36,10 @@ exports.selectUsers = async () => {
   return result.rows;
 }
 
+exports.selectComments = async (article_id) => {
+  const queryStr = "SELECT * FROM comments WHERE article_id = $1"
+  let queryValues = [article_id]
+  const result = await db.query(queryStr, queryValues)
+  if (!result.rows[0]) throw { status: 404, msg: "This article id does not exist !" }
```

Comment:
You are going to need to do another query to the database to check if the article exists. If the did exist but had no comments OR article didn't exist - we would be throwing an error here. We want to be consistent if the article does exist and just return the empty array